pkg/server/config/config.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	AuthenticatingProxy *AuthenticatingProxy `yaml:"auth_proxy"`
 	SSH                 SSH                  `yaml:"ssh"`
 	GitHub              GitHub               `yaml:"github"`
+	Metrics             Metrics              `yaml:"telemetry"`
```

Comment:
looks like the YAML is expecting `telemetry` but examples say `metrics`? I actually do like the name `telemetry` for the section in the config, but either way this should be consistent